src/main/resources/META-INF/plugin.xml

```diff
@@ -192,6 +193,8 @@
                  We can consider `VcsGroups` that seems to be more stable and place this action in VCS menu -->
             <add-to-group group-id="GitRepositoryActions" anchor="last"/>
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
+            <keyboard-shortcut first-keystroke="control alt shift M" keymap="Mac OS X" replace-all="true"/>
```

Comment:
Huh so the previous setting had no effect for Mac at all? 🤔 or was `control` interpreted as `cmd`?

Reply:
(went for that option indeed)